branchpro/apps/_inference.py

```diff
@@ -72,6 +75,30 @@ def __init__(self):
                         multiple=True  # Allow multiple files to be uploaded
                     ),
                     html.Div(id='incidence-data-upload'),
+                    html.H4([
+                        'You can upload your own serial interval here.'
+                    ]),
+                    dcc.Upload(
+                        id='upload-interval',
+                        children=html.Div([
+                            'Drag and Drop or ',
+                            html.A('Select Files',
+                                   style={'text-decoration': 'underline'}),
+                            ' to upload your Serial Interval.'
+                        ]),
+                        style={
```

Comment:
Let's save this rather long style dictionary in a variable above, and then just refer to it the two times we use it.